### PR TITLE
zip-directory-and-export-its-path v1.0.0

### PR DESCRIPTION
**Pull request by danielmcconville**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/zip-directory-and-export-its-path/v1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```
Comment (koral--) [inline steps/zip-directory-and-export-its-path/v1.0.0/step.yml]: Neither git nor wget seems to be needed here. The only dependency is a `zip`.

Comment (danielmcconville) [reply steps/zip-directory-and-export-its-path/v1.0.0/step.yml]: Good point. Sorry I missed that, this is the first time I have had a go at uploading a step. That was just the defaults from the template.

I have pushed an update.

File: steps/zip-directory-and-export-its-path/v1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- include_directory: true
+  opts:
+    description: "Set to true to include the parent directory in the zip file, false
+      if you \njust want the files in the directory.\n"
+    is_expand: false
+    is_required: true
+    summary: If set to true, the zip file will include the parent directory.
+    title: Include the directory in the zip
+    value_options:
+    - true
```
Comment (BirmacherAkos) [inline steps/zip-directory-and-export-its-path/v1.0.0/step.yml]: Please change the value options from bool to string:
```
value_options:
    - "true"
    - "false"
```

**Comment by BirmacherAkos** [comment]
Hi @danielmcconville!

Thanks for the step 🥇 
Could you please check my comment?

---

 - If you are done with the fix, please create a new release with the format 1.0.1 [Version naming convention](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#version-naming-convention)
(please create a patch release and don't modify the current one).
 - And create a new Pull Request for the new 1.0.1 version.

**Comment by BirmacherAkos** [comment]
Hi @koral-- !

Thanks again for your help 👍

File: steps/zip-directory-and-export-its-path/v1.0.0/step.yml (hunk 2 above)
Comment (danielmcconville) [reply steps/zip-directory-and-export-its-path/v1.0.0/step.yml]: Will do, thanks.

**Comment by danielmcconville** [comment]
> Hi @danielmcconville!
> 
> Thanks for the step 🥇
> Could you please check my comment?
> 
> * If you are done with the fix, please create a new release with the format 1.0.1 [Version naming convention](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#version-naming-convention)
>   (please create a patch release and don't modify the current one).
> * And create a new Pull Request for the new 1.0.1 version.

No problem. I thought that as the pull request hasn't been accepted there technically wasn't a version in existence, which is why I didn't create a new release with a new version number.

Happy to do that though.

Thanks!

**Pull request closed without merging**